src/main/webapp/tailwind.css

```
@@ -62,6 +62,7 @@
 @source './app/atlas/shared/standardized-competencies';
 @source './app/atlas/manage/import-standardized-competencies';
 @source './app/exercise/feedback';
+@source './app/exercise/exercise-headers/exercise-headers-information/result-history-dropdown/result-history-dropdown.component.html';
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)